src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div class="vocab input-field" :class="inputFieldClasses">
+    <!-- Attach label with ID when using -->
+    <input
+      v-bind="$attrs"
+      class="field"
+      :class="fieldClasses"
+      :disabled="isDisabled"
+      :readonly="isReadOnly"
+      :size="charCount"
+      @input="emitInput">
+
+    <div
+      v-if="hasLeftAddons"
+      class="left addons"
+      :class="leftAddonClasses">
+      <!-- @slot Left-side addons go here -->
+      <slot name="leftAddons">
+        <FontAwesomeIcon
+          v-if="iconSet[0]"
+          :icon="['fas', iconSet[0]]"
+          fixed-width/>
+      </slot>
+    </div>
+
+    <div
+      v-if="hasRightAddons"
+      class="right addons"
+      :class="rightAddonClasses">
+      <!-- @slot Right-side addons go here -->
+      <slot name="rightAddons">
+        <FontAwesomeIcon
+          v-if="iconSet[1]"
+          :icon="['fas', iconSet[1]]"
+          fixed-width/>
+      </slot>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { library } from '@fortawesome/fontawesome-svg-core'
+  import { faKeyboard } from '@fortawesome/free-solid-svg-icons'
+  import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+  import Colored from '@/mixins/colored'
+  import Indicating from '@/mixins/indicating'
+  import Joined from '@/mixins/joined'
+  import Rounded from '@/mixins/rounded'
+  import Scaled from '@/mixins/scaled'
+  import Simplified from '@/mixins/simplified'
+
+  import Invertible from '@/mixins/invertible'
+  import Unactionable from '@/mixins/unactionable'
+
+  library.add(faKeyboard)
+
+  /**
+   * ### Inputs fields accept user input, duh!
+   *
+   * Input fields form the basis of forms, the primary method of receiving input
+   * from the user, be it text, email addresses, passwords, dates or numbers.
+   */
+  export default {
+    name: 'InputField',
```

Comment:
This name will cause conflicts with the actual `InputField` component. Use `DoubleSlider` or even better `RangeField` (to comply with the convention of naming input fields with the 'Field` suffix).